packages/react-instantsearch/src/widgets/Snippet.js

```diff
@@ -15,29 +15,24 @@ import SnippetComponent from '../components/Snippet.js';
  * @propType {string} [tagName='em'] - the tag to be used for highlighted parts of the attribute
  * @example
  * import React from 'react';
- *
- * import { connectHits, Snippet, InstantSearch } from 'react-instantsearch/dom';
- *
- * const CustomHits = connectHits(({ hits }) =>
- * <div>
- *   {hits.map(hit =>
- *     <p key={hit.objectID}>
- *       <Snippet attributeName="description" hit={hit} />
- *     </p>
- *   )}
- * </div>
- * );
- *
+ * import { Snippet, InstantSearch, Hits } from "react-instantsearch/dom";
+ * 
  * export default function App() {
- *  return (
- *    <InstantSearch
+ *   return (
+ *     <InstantSearch
  *       appId="latency"
  *       apiKey="6be0576ff61c053d5f9a3225e2a90f76"
  *       indexName="ikea"
  *     >
- *       <CustomHits />
+ *       <Hits
+ *         hitComponent={hit => (
```

Comment:
The parameter hit are the props of the component, so you can't access it directly. You should use destructuring:

```jsx
<Hits
  hitComponent={({ hit }) => (
    ...
  )}
/>
```